File: Back/tpback/src/main/java/edu/trabajopracticointegrador/entities/Cost.java
package edu.trabajopracticointegrador.entities;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import com.fasterxml.jackson.annotation.JsonProperty;

@Entity
@Table(name="Cost")
public class Cost {
	
	@Id
	@GeneratedValue(strategy=GenerationType.AUTO)
	private @JsonProperty  int Id;
	@Column
	private  @JsonProperty  String Description;
	@Column
	private @JsonProperty  Boolean Active;
	
	
	
	public int getId() {
		return Id;
	}
	public void setId(int id) {
		Id = id;
	}
	public String getDescription() {
		return Description;
	}
	public void setDescription(String description) {
		this.Description = description;
	}
	public Boolean getActive() {
		return Active;
	}
	public void setActive(Boolean active) {
		Active = active;
	}

}

File: Front/webangular/trabajointegrador/src/app/model/cost.ts
export class Cost {
    public Id: number;
    public Description: string;
    public Active: boolean;
    constructor(id: number, description: string, active: boolean ){
            this.Id = id;
            this.Description = description;
            this.Active =  active;
              }
   public getId(): number {
            return this.Id;
        }
    public getDescription() {
            return this.Description;
        }
    public getActive() {
            return this.Active;
        }
    }
